subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiClasspathProvider.groovy

```diff
@@ -54,6 +54,7 @@ trait ToolingApiClasspathProvider {
         sharedSpec.allowPackage('junit.framework')
         sharedSpec.allowPackage('groovy')
         sharedSpec.allowPackage('org.codehaus.groovy')
+        sharedSpec.allowPackage('org.apache.commons')
```

Comment:
:x: I think it would be good to avoid allowing more external packages. The slimmer we can keep the Tooling API the better. It's really just for the use of a single method: `StringUtils.countMatches`.

Reply:
Isn't this just in tests? Will it affect Tooling API jars?

Reply:
No, it won't affect the TAPI jars, you're right @blindpirate . However I agree that it would be better to avoid the use of `StringUtils`, as we would expose something we don't need in general, so there's a risk of introducing unnoticed regressions in the future.

Reply:
Looks like Groovy has a [count](http://docs.groovy-lang.org/docs/groovy-1.7.3/html/groovy-jdk/java/lang/String.html#count(java.lang.String)), I should use that instead.